sdk/servicebus/service-bus/src/util/errors.ts

```diff
@@ -190,10 +192,51 @@ export function throwTypeErrorIfParameterIsEmptyString(
 /**
  * @internal
  * @ignore
- * Gets error message for when an operation is not supported in ReceiveAndDelete mode
- * @param failedToDo A string to add to the placeholder in the error message. Denotes the action
- * that is not supported in ReceiveAndDelete mode
+ * The error message for operations on the receiver that are invalid for a message received in receiveAndDelete mode.
  */
-export function getErrorMessageNotSupportedInReceiveAndDeleteMode(failedToDo: string): string {
-  return `Failed to ${failedToDo} as the operation is only supported in 'PeekLock' receive mode.`;
+export const InvalidOperationInReceiveAndDeleteMode =
+  "The operation is not supported in 'receiveAndDelete' receive mode.";
+
+/**
+ * @internal
+ * @ignore
+ * The error message for operations on the receiver that are invalid for a peeked message.
+ */
+export const InvalidOperationForPeekedMessage =
+  "This operation is not supported for peeked messages. Only messages received using 'receiveMessages()', 'subscribe()' and 'getMessageIterator()' methods on the receiver in 'peekLock' receive mode can be settled.";
+
+/**
+ * @internal
+ * @ignore
+ * The error message for when one attempts to settle an already settled message.
+ */
+export const MessageAlreadySettled = "The message has either been deleted or already settled";
```

Comment:
NIT
```suggestion
export const MessageAlreadySettled = "The message has either been deleted or already settled.";
```

Reply:
I'll do this as part of the pass on error messages before closing #12312 in a separate PR